codespell_lib/data/dictionary.txt

```diff
@@ -21613,8 +21677,14 @@ politicans->politicians
 politicing->politicking
 pollenate->pollinate
 polltry->poultry
+polocies->policies
+polocy->policy
+polocys->policys
```

Comment:
Suggested `policys` should be `policies`
Also add misspell `policys->policies, polices,` ?

Reply:
Yep, `policys` is not what I meant.  I'll fix that and add your other suggestion 👍🏻

Reply:
I added `police` rather than `polices` because `police` is normally used for singular and plural — wdyt?

Reply:
> I added police rather than polices.

Good for me, suggest `police` then.
I suggested `polices` because it just happen to be a valid word in aspell, hunspell, ispell, wordlist